docs/getting-started/integration.md

```diff
@@ -55,7 +35,7 @@ require(['path/to/chartjs/dist/Chart.min.js'], function(Chart){
 });
 ```
 
-**Note:** starting v2.8, Moment.js is an optional dependency for `Chart.js` and `Chart.min.js`. In order to use the time scale with Moment.js, you need to make sure Moment.js is fully loaded **before** requiring Chart.js. You can either use a shim:
+**Note:** starting v2.8, in order to use the time scale, you need to make sure [one of the available date adapters](https://github.com/chartjs/awesome#adapters) and corresponding date library are fully loaded **before** requiring Chart.js. The date adapter for Moment.js is included with Chart.js, but you still need to include Moment.js itself if this is the date adapter you choose to use. You can either use a shim:
```

Comment:
nit, should say v3

Reply:
Yeah, I debated what to do here. The version number was actually introduced with the 2.8 docs and you do need to do this if you use require.js with 2.8 or 2.9. Maybe we should just remove the version number? I'm not sure it adds that much. We don't talk about old versions anywhere else in the docs

Reply:
Removing the version number works for me

Reply:
done